### ISSUE TITLE
Elevated circulating suberic acid concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6023]
**Preferred term label:** Elevated circulating suberic acid concentration

**Synonyms** Elevated circulating octanedioic acid concentration

**Definition (free text, please give PubMed ID)** Concentration of suberic acid in the blood is above the normal range., PMID: 31842810

**Parent term (use hpo.jax.org/app)** Dicarboxylic acidemia HP:0040145

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
adding term as HP_0033177
